clients/ui/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+services:
+  frontend:
+    build: ./frontend
```

Comment:
This is only a suggestion/tip, I had a workaround for development (if we end up deciding some docker composes for dev, although we should work with clusters asap) that was the following:

```
  nginx:
    container_name: nginx_container
    restart: always
    image: nginx:XXXX
    volumes:
      - ./frontend/public:/usr/share/nginx/html
```
with this, there was a "hot reloading" method (which we added in makefile) so when executed it would build the frontend and since the volume was shared between the container and the machine you'll get the changes without restarting.

Reply:
That makes sense, nothing is stopping someone from adding that line to a docker-compose, I guess we could have a couple of example files - a more "production" style example and a development one using the bind mount to hot reload the frontend?

Reply:
nah don't worry, I just added as trivia, we can use the regular method which is more standard.